### PR TITLE
Upcoming fork status page

### PR DESCRIPTION
**Pull request by ziogaschr**

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -4,6 +4,14 @@
       <v-btn icon nuxt to="/" class="mr-1">
         <v-icon>mdi-home</v-icon>
       </v-btn>
+
```
Comment (meowsbits) [inline layouts/default.vue:7]: Is this pending something or dead code?

Comment (ziogaschr) [reply layouts/default.vue:7]: Yes, this is the button/link at the header for this page. I hide it for now as the rest functionality is already deployed on live

**Comment by ziogaschr** [comment]
Not on this PR. I was thinking of some stuff for the API of it :)

File: store/nodes.js [hunk 2]
```diff
@@ -21,11 +21,30 @@ export const mutations = {
     state.raw = raw
   },
   SET_CLIENTS(state, clients) {
+    // merge items with count > 10
```
Comment (meowsbits) [inline store/nodes.js:24]: Why 10 and not 2?

**Pull request merged**